tools/wpt/run.py

```diff
@@ -409,19 +409,34 @@ def setup_kwargs(self, kwargs):
             device = mozdevice.ADBDeviceFactory(adb=kwargs["adb_binary"],
                                                 device=device_serial)
             self._logcat.start(device_serial)
+            max_retries = 5
+            retry_count = 0
```

Comment:
I guess the `retry_count` is not really required? It's not used anywhere in the diff

Reply:
Good catch, thanks, that's from an earlier version with a `while True` loop that made me nervous :) I'll remove it.